### ISSUE TITLE
Get balance for contract address

### ISSUE THREAD

**Issue by jevonearth** [ecadlabs/go-tezos#3]
Implement a method to fetch the balance for a "contract" or "pkh".

Pay attention to data encoding, and use the appropriate decimal data type in go. Looking at http://tezos.gitlab.io/mainnet/tutorials/data_encoding.html might be helpful.

RPC documentation:

http://tezos.gitlab.io/mainnet/api/rpc.html#get-block-id-context-contracts-contract-id-balance

curl RPC examples;

`curl localhost:8732/chains/{chain_id}/blocks/{block_id}/context/delegates/{pkh}/balance`

`chain_id` is a symbolic chain identifier, e.g. main or test, or their equivalent Base58-encoded hash, e. g. NetXzGDuYoAawjv.
`block_id` is a symbolic block identifier, e.g. head of genesis, or any Base58-encoded block hash, e.g.

`curl -v localhost:8732/chains/main/blocks/head/context/delegates/tz3WXYtyDUNL91qfiCJtVUX746QpNv5i5ve5/balance`

Error for querying a non existent address returns http status 500 with a json payload as follows.  
```
[{"kind":"permanent","id":"proto.002-PsYLVpVv.context.storage_error","missing_key":["contracts","index","ed25519","8f","b5","ce","a6","2d","147c696afd9a93dbce962f4c8a9c91","balance"],"function":"get"}]
```

**Comment by jevonearth** [comment]
New methods have been added in 87fa9052ecbf62bdd00cff65e12a3514263d9440

**Comment by jevonearth** [comment]
Implemented in 87fa9052ecbf62bdd00cff65e12a3514263d9440
